client/src/components/LookForm.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import LookTile from "./LookTile.js";
+
+const LookForm = (props) => {
+  let aura;
+  if (props.look.aura) {
+    const auraOptions = props.look.aura.map((aura, index) => {
+      return (
+        <LookTile
+          key={index + "aura"}
+          id={index + "aura"}
+          name="aura"
+          look={aura}
+          handleInputChangeLook={props.handleInputChangeLook}
+        />
+      );
+    });
+    aura = <div>Their aura: {auraOptions}</div>;
+  }
+
+  let body;
+  if (props.look.body) {
+    const bodyOptions = props.look.body.map((body, index) => {
+      return (
+        <LookTile
+          key={index + "body"}
+          id={index + "body"}
+          name="body"
+          look={body}
+          handleInputChangeLook={props.handleInputChangeLook}
+        />
+      );
+    });
+    body = <div>Their body: {bodyOptions}</div>;
+  }
+
+  let clothes;
+  if (props.look.clothes) {
+    const clothesOptions = props.look.clothes.map((clothes, index) => {
+      return (
+        <LookTile
+          key={index + "clothes"}
+          id={index + "clothes"}
+          name="clothes"
+          look={clothes}
+          handleInputChangeLook={props.handleInputChangeLook}
+        />
+      );
+    });
+    clothes = <div>Their clothes: {clothesOptions}</div>;
```

Comment:
generally these elements should exist in the JSX at the bottom of the component, not as a part of the `clothesOptions` or `lookOptions`. I would move them down there

Reply:
Different classes can have different look components (Flake has clothes and eyes; Monstrous has aura and body, etc). I did this as a work around so IF clothes exists it will be rendered, but if it doesn't, there won't be some floating tags that you can't fill out.